src/main/java/uk/gov/ch/controller/update/OverseasEntityDataController.java

```diff
@@ -33,8 +33,8 @@ public ResponseEntity<OverseasEntityDataJson> getEntityEmail(@PathVariable("comp
 
         try {
             return ResponseEntity.ok(entityDataService.getEntityEmail(companyNumber));
-        } catch (OverseasEntityEmailAddressNotFoundException e) {
-            LOGGER.errorContext("The overseas entity email address could not be found for: " + companyNumber, e, dataMap.getLogMap());
+        } catch (CorporateBodyDetailsEmailAddressNotFoundException e) {
+            LOGGER.errorContext("The corporate body details email address could not be found for: " + companyNumber, e, dataMap.getLogMap());
```

Comment:
I'd be inclined to leave the wording of this error message as is, simply because this code is Overseas Entity specific.

Reply:
OK changed the log message back to ROE specific wording